magento/magento2
# Fix Issue #19872 - checking if image is in media directory

### Description (*)
Issue #19872 (closed) is still not fixed on develop branch.
On develop when image is added from media gallery while configuring category, it's path is saved as **/media/path_to_file**, but while Magento tries to check if this image is placed in media it compares it's path with **pub/media/** instead of **media/**.

### Fixed Issues (if relevant)
1. magento/magento2#19872: Magento 2.3 category edit page "Select from gallery" button not working.
2. magento/magento2#22092

### Manual testing scenarios (*)

1. Go to category add/edit form
2. Edit **Category Image** attribute using **Select from Gallery** option
3. Save category
4. Check if **Category Image** is visible in backoffice for this category

### Contribution checklist (*)
